Source/CesiumRuntime/Public/CesiumMetadataPrimitive.h

```diff
@@ -37,21 +46,24 @@ struct CESIUMRUNTIME_API FCesiumMetadataPrimitive {
    * @param model The model that stores EXT_feature_metadata extension
    * @param primitive The mesh primitive that stores EXT_feature_metadata
    * extension
-   * @param metadata The EXT_feature_metadata of the whole gltf
-   * @param primitiveMetadata The EXT_feature_metadata of the gltf mesh
+   * @param metadata The EXT_feature_metadata of the gltf mesh primitive.
    * primitive
    */
   FCesiumMetadataPrimitive(
       const CesiumGltf::Model& model,
       const CesiumGltf::MeshPrimitive& primitive,
-      const CesiumGltf::ExtensionModelExtFeatureMetadata& metadata,
-      const CesiumGltf::ExtensionMeshPrimitiveExtFeatureMetadata&
-          primitiveMetadata);
+      const CesiumGltf::ExtensionMeshPrimitiveExtFeatureMetadata& metadata,
+      const CesiumGltf::ExtensionModelExtFeatureMetadata& modelMetadata);
 
 private:
-  TArray<FCesiumMetadataFeatureTable> _featureTables;
+  TArray<FCesiumFeatureIdAttribute> _featureIdAttributes;
```

Comment:
The doc for the `FCesiumMetadataPrimitive` class says "It holds a collection of feature tables." That's not true anymore and should be updated.